target/ppc/int_helper.c

```diff
@@ -927,6 +927,21 @@ static inline bool get_acc(uint32_t flags)
     return flags & 0x1;
 }
 
+static inline uint8_t get_xmsk(uint32_t packed_masks)
```

Comment:
Move these 3 functions to internal.h? And also prefix them with 'ger_', since they're 100% specific to the ger instructions? (ger_get_xmsk, etc.)